app/app.html

```diff
@@ -14,7 +17,6 @@
         }
       }());
     </script>
-    <script type="text/javascript" src="http://media.twiliocdn.com/sdk/js/client/v1.3/twilio.min.js"></script>
```

Comment:
@leesander1 If this is removed it actually doesn't work, I did add Twilio into npm packages, however I have a feeling that webpack is not adding it to the actual product. So we can actually remove the Twilio npm package and just use the javascript link.